.github/workflows/reusable-build-provider.yaml

```diff
@@ -63,6 +63,15 @@ jobs:
           earthly +ci --SECURITY_SCANS=false --IMAGE=ttl.sh/kairos-${{ inputs.flavor }}-${{ github.sha }}-provider:24h --FLAVOR=${{ inputs.flavor }} --VARIANT=standard -K3S_VERSION=latest
           sudo mv build/* .
           sudo rm -rf build
+      - name: Extract packages versions
+        run: |
+          earthly +framework-versions --IMAGE=ttl.sh/kairos-${{ inputs.flavor }}-${{ github.sha }}-provider:24h --FLAVOR=${{ inputs.flavor }} --VARIANT=standard -K3S_VERSION=latest
```

Comment:
`framework` target (or `framework-versions`) has no need for `IMAGE` , `VARIANT` nor `K3S_VERSION` to be set. You can remove these. You only need to set `FLAVOR`.

Reply:
BTW I dont even know if it works yet, and if it does I think it could be part of the +ci or +all targets, especially because on the provider we have to generate it later.

Reply:
It could be an artifact of `base-image` like `build/IMAGE` and `build/VERSION` so that we don't have to run the framework-versions target explicitly (we may not even need a new target then)

Reply:
problem is, it needs to be runs after the framework stuff is installed as it requires the luet database to be intact but just after that we remove the database and all temp files, so it needs to be done twice for the provider I think, first ont eh base framework after just installing and then in the provider after installing k3s. And then on provider it needs to be merged together with the normal base versions xD

Reply:
> It could be an artifact of `base-image` like `build/IMAGE` and `build/VERSION` so that we don't have to run the framework-versions target explicitly (we may not even need a new target then)

good one, I used that for the isos and the framework for the images so both have their versions included. Plus delayed the database removal so we can gather all the provider and k3s artifact versions !